### PR TITLE
Fixed live styles

### PR DESCRIPTION
**Pull request by mbrandonw**
This is just for posterity, but #104 messed with some nav bar styles on our live streaming controllers and this fixes that. Will merge when tests pass.

### CODE REVIEW

File: Library/Styles/BaseStyles.swift [hunk 1]
```diff
@@ -119,5 +125,12 @@ private let baseNavigationBarStyle =
     NSForegroundColorAttributeName: UIColor.black,
     NSFontAttributeName: UIFont.ksr_callout()
     ]
-    <> UINavigationBar.lens.translucent .~ false
```
Comment (theginbin) [inline Library/Styles/BaseStyles.swift]: quick q, the default `translucent` value is `true`. how come this was removed?

Comment (mbrandonw) [inline Library/Styles/BaseStyles.swift]: ah sorry, i thought the default was `false` and it seemed to work, but now i see it's wrong. fixing...

File: Kickstarter-iOS/Views/Storyboards/Activity.storyboard [hunk 2]
```diff
@@ -385,7 +385,7 @@
                                                         <constraint firstAttribute="width" constant="80" id="vCB-Xb-z6f"/>
                                                     </constraints>
                                                 </imageView>
-                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="HERTZFELDT ON BLU-RAY" textAlignment="natural" lineBreakMode="tailTruncation" numberOfLines="2" baselineAdjustment="alignBaselines" minimumScaleFactor="0.69999999999999996" translatesAutoresizingMaskIntoConstraints="NO" id="peK-ts-qtm" customClass="StyledLabel" customModule="Library">
```
Comment (mbrandonw) [inline Kickstarter-iOS/Views/Storyboards/Activity.storyboard]: sorry saw this `StyledLabel` getting logged to the console so wanted to remove. unrelated to this pr but still nice to get rid of

Comment (theginbin) [inline Kickstarter-iOS/Views/Storyboards/Activity.storyboard]: oh nice, thanks!

File: Library/Styles/BaseStyles.swift (hunk 1 above)
Comment (theginbin) [inline Library/Styles/BaseStyles.swift]: yeah apple loves that translucent look

**Pull request merged**